app/views/mailing_list/steps/_already_qualified.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>We're sorry, but we only provide tailored guidance if you're interested in training to teach</h1>
```

Comment:
@ethax-ross any chance we can make this span a bit wider to match the width of the copy underneath?